package.json

```diff
@@ -53,14 +72,17 @@
     "@types/node": "17.0.21",
     "@typescript-eslint/eslint-plugin": "5.13.0",
     "@typescript-eslint/parser": "5.10.2",
+    "cypress": "^9.1.0",
     "eslint": "8.10.0",
     "eslint-config-prettier": "8.4.0",
+    "eslint-plugin-cypress": "^2.10.3",
     "jest": "27.5.1",
+    "jest-preset-angular": "11.1.1",
+    "nx": "13.9.5",
     "prettier": "2.5.1",
     "prisma": "^3.10.0",
     "ts-jest": "27.1.3",
-    "typescript": "4.6.2",
-    "nx": "13.9.5"
+    "typescript": "^4.5.2"
```

Comment:
One of the things I dislike, when using nx<>angular -> downgrade or pinning the typescript version.
(But this is the way how to do this 👍🏽 )

Reply:
:yea: 😠